source/agora/common/BlockStorage.d

```diff
@@ -781,5 +781,5 @@ version(none) unittest
         assert(random_block.header.merkle_root == merkle_roots[idx]);
     }
 
-    other.release();
+    other.unMap();
```

Comment:
Sorry, I noticed that after. I thought `release` was unused and so removed it. But it is a better name, all things considered, so the integration test PR fixes this.